pkg/eval/math/math.go

```diff
@@ -285,6 +285,50 @@ import (
 // ▶ (float64 NaN)
 // ```
 
+//elvdoc:fn max
+//
+// ```elvish
+// math:max $number...
+// ```
+//
+// Outputs the maximum number in the arguments. If there are no arguments
+// an exception is thrown. If any number is NaN then NaN is output.
+//
+// Examples:
+//
+// ```elvish-transcript
+// ~> math:max
+// ▶ ?(WTF)
```

Comment:
Replace this with an actual transcript of the exception.

Reply:
Argh! Yep, I meant that as a placeholder until I confirmed the exact spelling of the exception. Fixed.